src/Writer/CsvWriter.php

```diff
@@ -126,6 +126,12 @@ public function write(array $data): void
             ++$this->position;
         }
 
+        if (1 !== \strlen($this->delimiter) ||
+            1 !== \strlen($this->enclosure) ||
+            1 !== \strlen($this->escape)) {
+            throw new InvalidDataFormatException();
+        }
```

Comment:
The fputcsv only accepts one character for `delimiter` `enclosure` and `escape`.

Previous php versions would just accept more and return null (not sure if due to the supress operator or always), but php8 throws an exception instead.

To avoid behavioral bc breaks, I just add this check before calling the function. (we could could consider not allowing to do a new `CsvWriter` with more than one char on next major).

Reply:
Can you please make this info part of the commit message? Thanks